src/test/resources/inspections/nullableUnequalTypes.kt

```diff
@@ -0,0 +1,14 @@
+package inspections
+
+import io.kotest.core.script.test
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+
+class NullableUnequalTypesTest : FunSpec({
+
+   test("foo") {
+      val x: Int? = null
+      x shouldBe 5
+      5 shouldBe x
```

Comment:
comparing Int? and Int should not cause warnings

Reply:
Yeah I think that's fine.